adapters/repos/db/index_integration_test.go

```diff
@@ -58,7 +58,7 @@ func TestIndex_DropIndex(t *testing.T) {
 	}()
 	testClassName := "deletetest"
 	index, err := NewIndex(IndexConfig{
-		RootPath: dirName, Kind: kind.Thing, ClassName: schema.ClassName(testClassName),
+		RootPath: dirName, Kind: kind.Object, ClassName: schema.ClassName(testClassName),
```

Comment:
similar to above, I think we could remove `Kind` from `IndexConfig` - not required as of this PR, we can create a follow up issue